code/game/machinery/computer/extraction_cargo.dm

```diff
@@ -49,9 +49,13 @@
 		new /datum/data/extraction_cargo("Sal-Acid Medi-Pen ",			/obj/item/reagent_containers/hypospray/medipen/salacid,				50, CAT_MEDICAL) = 1,
 		new /datum/data/extraction_cargo("Mental-Stabilizer Medi-Pen ",	/obj/item/reagent_containers/hypospray/medipen/mental,				50, CAT_MEDICAL) = 1,
 		new /datum/data/extraction_cargo("Standard First-Aid Kit ",		/obj/item/storage/firstaid/regular,									250, CAT_MEDICAL) = 1,
+		new /datum/data/extraction_cargo("C-Fear Protection Injector ",	/obj/item/lc13_clerk_fear_immunity_injector,						300, CAT_MEDICAL) = 1,
```

Comment:
Move to Equipment, medical is for things that directly heal you, equipment is for upgrades

Reply:
It's an injector. Makes more sense in the medical section.